modules/date.js

```diff
@@ -0,0 +1,12 @@
+import { DateTime } from './luxon.min.js';
+
+const dateTime = document.getElementById('dateTime');
+
+const setInitialTime = () => {
+  const dt = DateTime.now();
+  const getCurrentTime = () => `${dt.hour}:${dt.minute}:${dt.second}`;
+  const f = { month: 'long', day: 'numeric' };
+  dateTime.innerHTML = `${dt.setLocale('en-US').toLocaleString(f)} ${dt.year}, <span>${getCurrentTime()}</span>`;
+};
+
+export default setInitialTime;
```

Comment:
- [ ] Your date is only displayed on the contact page. Kindly note that the date should always be displayed under the navigation bar, regardless of the page.

- [ ] Your date is also not well displayed on the contact page. It appears to be covered by something. Kindly fix it.

  list page | add new page | contact section.
  :---:|:---:|:---:
  ![image](https://github.com/yayaismaail/awesome-books-ES6/assets/36542986/97923b43-9781-4ce4-bea3-1ffd0c0d3ee8) | ![image](https://github.com/yayaismaail/awesome-books-ES6/assets/36542986/3800ad01-4e9a-4285-b51e-b3388c806384) | ![image](https://github.com/yayaismaail/awesome-books-ES6/assets/36542986/1a73eea0-25d9-40a5-9da2-d4b1f973ae3a)